<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="MLIVE-LP00960" ignored="0" name="Command line test" tests="4" failures="1" timestamp="23 Jan 2019 20:29:56 GMT" time="87.672" errors="0">
  <testcase name="changePasswdTest" time="0.018" classname="com.qtpselenium.facebook.pom.testcases.ChangePasswordTest">
    <skipped/>
  </testcase> <!-- changePasswdTest -->
  <testcase name="doLogin" time="23.393" classname="com.qtpselenium.facebook.pom.testcases.LoginTest"/>
  <testcase name="doLogin" time="35.148" classname="com.qtpselenium.facebook.pom.testcases.LoginTest">
    <failure type="java.lang.AssertionError" message="failure message">
      <![CDATA[java.lang.AssertionError: failure message
	at com.qtpselenium.facebook.pom.base.BaseTest.reportFailure(BaseTest.java:88)
	at com.qtpselenium.facebook.pom.testcases.LoginTest.doLogin(LoginTest.java:41)
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- doLogin -->
  <testcase name="testProfile" time="19.927" classname="com.qtpselenium.facebook.pom.testcases.ProfileTest"/>
</testsuite> <!-- Command line test -->
